etc/docker/Dockerfile.debian.sid

```diff
@@ -22,6 +22,7 @@ RUN apt-get -y --no-install-recommends install \
     libgrpc++-dev \
     libprotobuf-dev \
     libssl-dev \
+    libxtensor-dev \
```

Comment:
This is available in brew for macos. For `bookworm` this will be a builtin, but we will need to vendor xtensor for other platofrms.

Reply:
Also, we can _potentially_ drop the xtensor requirement and look into other options (maybe something already in boost). I originally picked xtensor because it was going to allow me to sew together multiple independent allocations under one tensor view with the `xchunked_array`. I thought I was going to need that at one point, but that was due to a misunderstanding of mine about how the tensor data was arriving over the wire. So we may be able to use something simpler. On the other hand, `xtensor` is pretty nice.